.gitignore

```diff
@@ -19,3 +19,5 @@ TestServer/wwwroot/js/
 Pods/
 Package.resolved
 .build/
+# This can be generated using: swift package generate-xcodeproj
```

Comment:
As a new user/contributor - do I have to do this if I just clone the repo? What are the advantages of ignoring this vs. having this checked in? How is it different from SignalRClient.xcodeproj? I guess this has something to do with CocoaPods vs SPM.

Reply:
This can be checked in. The .xcodeproj file isn't really needed in the base directory. The project file still is available in the Examples directory along with the Xcode workspace. This was probably added along with the 'swift package init' command.